spec/car_spec.rb

```diff
@@ -37,4 +37,16 @@
 
     expect(car.total_cost).to eq(54000)
   end
+
+  it 'has a total cost' do
```

Comment:
You want to make sure you are updating the tests' names when creating new ones rather than keeping what's left on copy & paste
```suggestion
  it 'has a color' do
```